packages/cspell-lib/src/SpellingDictionary/DictionaryLoader.test.ts

```diff
@@ -75,9 +75,9 @@ describe('Validate DictionaryLoader', () => {
             [sample('words.txt'), {}, 'apple'],
             [sample('words.txt'), { type: 'S' }, 'pear'],
             [sample('words.txt'), { type: 'C' }, 'strawberry'],
-            [dict('csharp.txt'), {}, 'const'],
-            [dict('csharp.txt'), { type: 'S' }, 'const'],
-            [dict('csharp.txt'), { type: 'C' }, 'const'],
+            [dict('node_modules/cspell-dict-csharp/csharp.txt.gz'), {}, 'const'],
+            [dict('node_modules/cspell-dict-csharp/csharp.txt.gz'), { type: 'S' }, 'const'],
+            [dict('node_modules/cspell-dict-csharp/csharp.txt.gz'), { type: 'C' }, 'const'],
```

Comment:
Since it isn't loading from the dictionary directory, removing call to `dict` will fix the test.

Reply:
OK, so just remove these 3 test lines?